File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/flickr/People.java

package com.lifedawn.bestweather.data.remote.retrofit.responses.flickr;

import java.io.Serializable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class People implements Serializable
{

    @SerializedName("haspeople")
    @Expose
    private Integer haspeople;
    private final static long serialVersionUID = -6712516251128857555L;

    public Integer getHaspeople() {
        return haspeople;
    }

    public void setHaspeople(Integer haspeople) {
        this.haspeople = haspeople;
    }

}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/flickr/People.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.flickr

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class People : Serializable {
    @SerializedName("haspeople") @Expose var haspeople: Int? = null

    companion object {
        private const val serialVersionUID = -6712516251128857555L
    }
}